### ISSUE TITLE
"myServer" limitation shows info from other domains

### ISSUE THREAD

**Issue by vitko-bg** [Ylianst/MeshCentral#1871]
On additional domains with applied options from here https://github.com/Ylianst/MeshCentral/issues/1862#issuecomment-701505785 on My Server tab is still visible server state and all agents information from other domians:

![image](https://user-images.githubusercontent.com/59362521/95136263-b9790080-075d-11eb-93ce-948770c904b8.png)

Also - on Stats(also info from agents on other domains):

![image](https://user-images.githubusercontent.com/59362521/95136296-cbf33a00-075d-11eb-8dc1-d3d4d856c3f3.png)

Can you please add also to "myServer" options "General": false and "Stats": false ???
Or entirely all Server tab to be "false"

Thanks.

**Comment by Ylianst** [comment]
Ok, I will add an option to remove the entire tab.

**Comment by Ylianst** [comment]
Just published MeshCentral v0.6.58. In this version instead of this:

```
"myServer": {
  "Backup": false,
  "Restore": false,
  "Upgrade": false,
  "ErrorLog": false,
  "Console": false,
  "Trace": false
}
```

Just put this:

```
"myServer": false,
```

It will remove the "My Server" tab completely. As with the other options, it's not just that the HTML removes the tab, but the server will not serve any of the information even if asked. Let me know if that works.

**Comment by vitko-bg** [comment]
Confirmed - tested and all is fine!!!
Thanks a lot!
